frontend/.env.development

```diff
@@ -1 +1,3 @@
-REACT_APP_FEEDBACK_BUTTON=false
\ No newline at end of file
+REACT_APP_FEEDBACK_BUTTON=false
+PORT=3000
+BROWSER=none
```

Comment:
`start:v2` run graduate on port 3002, not on 3000. BROWSER=none stops the browser from opening up when the frontend runs. Did this since it is confusing to have the browser open at port 3000 when the app actually runs on 3002.